src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -244,5 +244,16 @@ public void resetLastAngles() {
         m_rearRight.resetLastAngle();
     }
 
+    public double getRollValue() {
+        double[] ypr = new double[3];
+        m_pigeon.getYawPitchRoll(ypr);
+        return Math.abs(ypr[2]);
+    }
+    public double getRoll() {
+        return(m_pigeon.getRoll());
+    }
+    public double getYaw() {
+        return(m_pigeon.getYaw());
+    }
```

Comment:
All 3 of these methods need to be either displayed to Networktables, or we just do a System.out.println() so that we can read them and understand what the ROLL_TOLERANCE should be or whether we need to change from roll to pitch.

We could just add this to the periodic of the SwerveSubsystem():
`System.out.printf("RollValue: %.2f, Roll: %.2f, Pitch: %.2f", getRollValue(), getRoll(), getPitch());`

Also pitch is missing from these methods. Should have one which `returns m_pigeon.getPitch()`